src/jit/codegencommon.cpp

```diff
@@ -1509,8 +1511,6 @@ bool CodeGenInterface::genCodeIndirAddrNeedsReloc(size_t addr)
         return true;
     }
 
-    // Else jitting.
```

Comment:
Is this a formatting change, or just that this was a bad comment?
It could maybe say "Otherwise we are jitting."